packages/eslint-plugin/tests/rules/member-ordering.test.ts

```diff
@@ -1417,6 +1417,23 @@ class Foo {
         },
       ],
     },
+    {
+      code: `
+class Foo {
+    A: string;
+    constructor() {}
+    get B() {}
+    set B() {}
+    get C() {}
+    set C() {}
+    D(): void;
+}           `,
+      options: [
+        {
+          default: ['field', 'constructor', ['get', 'set'], 'method'],
```

Comment:
A few more test cases to consider:

* An empty array
* Array with just one member
* Array with three members

I'd be good to have more than just `'get`` and `'set'` tested too.

Reply:
Thanks for the review! I've added a few more test cases that should cover all those scenarios.